sematic/ui/src/pipelines/graph/RunNode.tsx

```diff
@@ -75,18 +78,19 @@ export default function RunNode(props: NodeProps) {
           borderColor: lighten(color.light, props.selected ? 0.5 : 0.5),
           backgroundColor: lighten(color.light, props.selected ? 0.7 : 0.9),
           color: color.dark,
+          ...hiddenNodePadding
           //"&:hover": {
           //  backgroundColor: lighten(color.light, props.selected ? 0.7 : 0.87),
           //},
           //display: "-webkit-inline-flex",
         }}
       >
         <AlertTitle>
-          <RunStateChip state={run.future_state} />
-          {run.name}
+          <RunStateChip runType={runType} state={run.future_state} />
```

Comment:
A single icon looks too different than `<RunStateChip>.` And many logics in `<RunStateChip>` looks necessary for rendering an icon. How about we render a new component in case of `runType==="hidden" `? We can probably replace the entire `<AlertTitle />`

Reply:
Sure I will make all these changes.